app/assets/javascripts/components/instructor_panel.js.jsx

```diff
@@ -7,6 +7,20 @@ var InstructorPanel = React.createClass({
       action: this.props.queuePop,
     };
   },
+  getQueuePopandPinButtonData: function () {
```

Comment:
Could you clarify the expected functionality when an instructor presses this button?

Reply:
When the instructor presses the Queue Pop and Pin Button, the user should be removed from the main requests queue and instead should be pinned by the instructor. Can also be used to pin multiple users at a time.

Reply:
👍 cool – thoughts on calling it "pin top" or something like that?